fjord/journal/models.py

```diff
@@ -16,6 +16,8 @@ class RecordManager(models.Manager):
     @classmethod
     def log(cls, type_, app, src, action, msg, instance=None, metadata=None):
         msg = msg.encode('utf-8')
+        if len(msg) > 255:
+            msg = msg[:255]
```

Comment:
If you're worried about unicode you could maybe use the django utils `truncate`? It probably handles it smartly.